spec/contracts/projects/shared_contract_examples.rb

```diff
@@ -107,8 +107,12 @@ def expect_valid(valid, symbols = {})
   context 'if the identifier is nil' do
     let(:project_identifier) { nil }
 
-    it 'is invalid' do
-      expect_valid(false, identifier: %i(blank))
+    it 'is replaced for new project' do
+      if project.new_record?
+        expect_valid(true)
+      else
+        expect_valid(false, identifier: %i(blank))
+      end
```

Comment:
This distinction between creating and updating a project should be in the create_contract_spec/update_contract_spec.

Reply:
Good point.Done